client/platform/desktop/backend/native/viame.ts

```diff
@@ -336,7 +336,7 @@ async function train(
       } catch (err) {
         console.error(err);
         exitCode = 1;
-        bodyText.unshift(err.toString('utf-8'));
+        bodyText.unshift((err as Error).toString());
```

Comment:
I removed it because it's a typescript error.  According to `toString` doesn't take arguments.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString